src/store/middleware/inputMiddleware.js

```diff
@@ -42,6 +42,14 @@ function updateDrive(prevComputedInput, computedInput, dispatch) {
       dispatch(requestDrive({ straight, steer }));
     }
   }
+  if (computedInput.drive.activeSuspension) {
```

Comment:
We shouldn't be checking this in an if statement. The above code checks `if(computedInput.drive.tank)` because `tank` is a boolean we've specifically added to the state. What this code does is just check if the suspension power is truthy, which isn't what we want. So you can just remove the if statement and put the code outside of it.